frontend/raybamusic/.editorconfig

```diff
@@ -0,0 +1,16 @@
+# Editor configuration, see https://editorconfig.org
```

Comment:
I think this file should not be part of the project, since it's specific for some editors and not all developers could need using it. Also, it has nothing to do with the project itself.

If you don't mind, please:

1. Remove this file from the commit of this PR.
2. Decide whether ignore this file in your own computer, locally (it is possible to configure git to achieve it) or just ignore this file from the .gitignore file of this repository (much better in a new commit and or PR) if you still need using it when you are developing.

Thank you!